src/navx_node.cpp

```diff
@@ -52,9 +60,11 @@ int main(int argc, char** argv) {
   pnh->param<bool>("publish_euler", publish_euler, false);
```

Comment:
Also, all of this should be at top-of-file documentation, where is it?

Reply:
The documentation is in the readme and the launch file. It doesn't need to be in a third place.